src/utils/parse-smart-query.ts

```diff
@@ -181,9 +201,8 @@ export const parseSmartQuery = (
                             andQuery.push({ [key]: { $nin: pureValue } })
                         }
                     } else {
-                        queryValue = term
-                            .split('IN')
-                            .map((x) => x.trim())[1]
+                        // TODO make , inside quotes work
```

Comment:
What is this TODO for ? Please create a ticket for it if needed

Reply:
just an unrelated issue I have noticed while working on this section of the code (created https://github.com/dataloop-ai/components/issues/943)